docs/CHANGELOG.rst

```diff
@@ -12,12 +12,17 @@ Unreleased
 
 Changed
 -------
+- **BACKWARD INCOMPATIBLE:** Refactor ``trimmomatic-single`` and ``trimmomatic-paired``
+  processes
 - Bump STAR aligner version in ``resolwebio/rnaseq`` docker image to 2.5.4b
 - Bump Primerclip version in ``resolwebio/dnaseq`` docker image
 
 Added
 -----
 - Add CNVKit, LoFreq and GATK to ``resolwebio/dnaseq`` docker image
+- **BACKWARD INCOMPATIBLE:** Merge ``align-bwa-trim`` and ``align-bwa-trim2``
+  process functionality. Retain only the refactored process under slug
+  ``align-bwa-trim``.
```

Comment:
Full stop removal should be done in this commit, not some further commit down the road.